src/notion-to-md.ts

```diff
@@ -20,9 +19,11 @@ import { getBlockChildren } from "./utils/notion";
 export class NotionToMarkdown {
   private notionClient: Client;
   private customTransformers: Record<string, CustomTransformer>;
+  private convertImagesToBase64: boolean;
```

Comment:
can you move this under the config option as per the latest changes in the master?

Reply:
yeah that sounds good.